chapters/7-publication.tex

```diff
@@ -0,0 +1,681 @@
+to the locations where they are placed in the associated published material,
+so ensure that the raw components of figures or tables are clearly identified.
+Documentation in the master script is often used to indicate this information.
+For example, outputs should clearly correspond by name to an exhibit in the paper, and vice versa.
+(Supplying a compiling {\LaTeX} document can support this.)
+Code and outputs which are not used should be removed before publication.
+
+\subsection{Releasing a reproducibility package}
+
+% finding a place to publish the package
+Once your data and code are polished for public release,
+all you need to do is find a place to publish your materials.
+This is slightly easier said than done,
+as there are a few variables to take into consideration
+and, at the time of writing, no global consensus on the best solution
+and there are a variety of archives and storage providers
+that cater to different needs.
+The technologies available are likely to change dramatically
+over the next few years;
+the specific solutions we mention here highlight some current approaches
+as well as their strengths and weaknesses.
+
+% code licenses
+Unlike data, code usually has few external constraints to publication.
+The research team owns the code in almost all cases,
+and code is unlikely to contain identifying information
+(though you must check carefully that it does not).
+Publishing code also requires assigning a license to it;
+in a majority of cases, code publishers like GitHub
+offer extremely permissive licensing options by default.
+If you do not provide a license, nobody can use your code.
+You may be required to put your materials into the public domain;
+if not, it is common to only require attribution and citation for their reuse,
+without putting any barriers or restrictions to accessing the code.
+
+% pros and cons of GitHub
+One option for releasing a reproducibility package is GitHub.
+Making a public GitHub repository is completely free.
+It can hold any file types,
+provide a structured, compressed download of your whole project,
+and allow others to look at alternate versions or histories easily.
+It is straightforward to simply upload a fixed directory to GitHub,
+apply a sharing license, and obtain a URL for the whole package.
+There is a strict size restriction of 100MB per file and
+a restriction on the size of the repository as a whole,
+so larger projects will need alternative solutions.
+However, GitHub is not the ideal platform to release reproducibility packages.
+It is built to version control code, and to facilitate collaboration on it.
+It is not an archive, meaning that it does not guarantee the permanence
+of uploaded materials, it does not guarantee the permanence of the access URL,
+and it does not manage citations or non-code licenses by default.
+
+Features to look for in a platform to release reproducibility packages
+and that are not offered by GitHub, include:
```

Comment:
Taking the GitHub mention out, as it could be argue that some of these features are available through GitHub

```suggestion
Features to look for in a platform to release reproducibility packages include:
```